apis/v1/httproute_types.go

```diff
@@ -361,6 +369,105 @@ type HTTPRouteTimeouts struct {
 	BackendRequest *Duration `json:"backendRequest,omitempty"`
 }
 
+// HTTPRouteRetry defines retry configuration for an HTTPRoute.
+//
+// Implementations SHOULD retry on connection errors (disconnect, reset, timeout,
+// TCP failure) if a retry stanza is configured.
```

Comment:
Any reason not to make this configurable?

Reply:
Primarily because there's _way_ too much variance in how different implementations define, default or bucket or even allow these to be configurable for this to be generally implementable within extended conformance, and secondarily because it's a simpler UX for what is often a good practice.

Reply:
FTR I think that makes sense, @mikemorris.